package.json

```diff
@@ -3,63 +3,48 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "ng:build:demo": "ng build packages --prod --base-href=https://ihym.github.io/ngx-timeago/",
-    "build:demo": "cpr dist/ngx-timeago demo/ngx-timeago && npm run ng:build:demo && rimraf demo/ngx-timeago",
-    "build:lib": "rimraf dist && ng-packagr -p lib/package.json && gulp",
-    "lint": "ng lint",
+    "build:demo": "ng build ngx-timeago-demo --base-href=https://ihym.github.io/ngx-timeago/",
+    "build:lib": "ng build ngx-timeago && npm run lib:compile:locales",
+    "lib:compile:locales": "tsc --project tsconfig.locales.json",
     "ng": "ng",
     "start": "ng serve",
-    "test": "ng test packages --watch=false",
-    "tdd": "ng test dev --watch",
+    "test": "ng test",
+    "tdd": "ng test --watch",
     "commit": "git-cz",
-    "analyze": "npm run build:lib && cpr dist/ngx-timeago demo/ngx-timeago && npm run ng:build:demo  -- --stats-json && rimraf demo/ngx-timeago && webpack-bundle-analyzer dist/demo/stats.json"
+    "analyze": "npm run build:lib && npm run build:demo  -- --stats-json && webpack-bundle-analyzer dist/demo/stats.json"
   },
   "dependencies": {
-    "@angular/animations": "10.0.8",
-    "@angular/cdk": "10.1.2",
-    "@angular/common": "10.0.8",
-    "@angular/compiler": "10.0.8",
-    "@angular/core": "10.0.8",
-    "@angular/forms": "10.0.8",
-    "@angular/material": "10.1.2",
-    "@angular/platform-browser": "10.0.8",
-    "@angular/platform-browser-dynamic": "10.0.8",
-    "@angular/router": "10.0.8",
-    "core-js": "3.6.5",
-    "rxjs": "6.6.2",
-    "zone.js": "0.10.3"
+    "@angular/animations": "^16.0.0",
+    "@angular/cdk": "^16.0.1",
+    "@angular/common": "^16.0.0",
+    "@angular/compiler": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@angular/forms": "^16.0.0",
+    "@angular/material": "^16.0.1",
+    "@angular/platform-browser": "^16.0.0",
+    "@angular/platform-browser-dynamic": "^16.0.0",
+    "@angular/router": "^16.0.0",
+    "bootstrap": "4.5.2",
+    "ngx-timeago": "file:dist/ngx-timeago",
```

Comment:
I guess this needs to be removed? Why not use ts paths as before?

Reply:
I think if we add the paths to the demo tsconfig it should work, will prepare a PR for this as well.